app.js

```diff
@@ -173,20 +156,22 @@ roundModal.btn.addEventListener('click', () => {
     roundModal.modal.classList.add('invisible');
 })
 
-function disablingBtn() {
-    playerElements.paperBtn.disabled = true;
-    playerElements.rockBtn.disabled = true;
-    playerElements.scissorsBtn.disabled = true;
+function switchBooleanValue() {
+    const btnsToClick = [
+        playerElements.paperBtn,
+        playerElements.rockBtn,
+        playerElements.scissorsBtn,
 
-    submitPlayerSelectionBtn.disabled = true;
-}
+        submitPlayerSelectionBtn
+    ]
 
-function btnActive() {
-    playerElements.paperBtn.disabled = false;
-    playerElements.rockBtn.disabled = false;
-    playerElements.scissorsBtn.disabled = false;
-
-    submitPlayerSelectionBtn.disabled = false;
+    btnsToClick.forEach(function(btnToClick) {
+        if(btnToClick.disabled === false) {
```

Comment:
nice function! however, you could make it shorter ;) 
```
btnsToClick.forEach(function(btnToClick) {
  btnToClick.disabled = !btnToClick.disabled
})
```
In final you want to switch to oposite current value.